metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/delete_test.clj

```diff
@@ -33,59 +33,46 @@
 (deftest delete-group-general
   (cd-spec/general-delete-test :access-group ["REG_PROV" "SMAL_PROV" "CMR"]))
 
+(defn- is-association-tombstone?
+  "Returns true if the latest revision of the given association is a tombstone."
+  [association]
+  (let [{:keys [status concept]} (util/get-concept-by-id (:concept-id association))]
+    ;; make sure the call succeeded
+    (is (= 200 status))
+    (:deleted concept)))
+
 (deftest tag-delete-cascades-associations
   (testing "delete cascades to tag associations"
     (let [tag-collection (util/create-and-save-collection "REG_PROV" 1)
           tag (util/create-and-save-tag 1)
-          tag-association (util/create-and-save-tag-association tag-collection tag 1 1)
-          tag-association-id (:concept-id tag-association)
-          {status :status tag-association-concept :concept} (util/get-concept-by-id tag-association-id)
-          _ (util/delete-concept (:concept-id tag))
-          {tombstone-status :status tombstone :concept} (util/get-concept-by-id tag-association-id)]
+          tag-association (util/create-and-save-tag-association tag-collection tag 1 1)]
       (testing "tag association was saved and is not a tombstone"
-        (is (= 200 status))
-        (is (not (:deleted tag-association-concept))))
+        (is (= false (is-association-tombstone? tag-association))))
       (testing "tag association is tombstoned after tag is deleted"
-        (is (= 200 tombstone-status))
-        (is (:deleted tombstone))))))
+        (util/delete-concept (:concept-id tag))
```

Comment:
I'm glad you made the change to do the deletion here within the testing block. I almost commented on it on the other PR, but held back. 😄